src/connectionController.ts

```diff
@@ -433,7 +435,17 @@ export default class ConnectionController {
     }
 
     log.info('Successfully connected');
-    void vscode.window.showInformationMessage('MongoDB connection successful.');
+
+    if (oldConnectionId) {
+      const currentConnectionName = oldConnectionId
+        ? this.getSavedConnectionName(oldConnectionId)
+        : 'previous server';
+      const nextConnectionName = this.getSavedConnectionName(connectionId);
+
+      void vscode.window.showInformationMessage(`Disconnected from ${currentConnectionName} and connected to ${nextConnectionName}.`);
+    } else {
+      void vscode.window.showInformationMessage('MongoDB connection successful.');
```

Comment:
I like the change in the new messages to use the connection name. Would it be possible to do the same in the existing message?
```suggestion
      const connectionName = this.getSavedConnectionName(connectionId);

      void vscode.window.showInformationMessage(connectionName ? `Connected to ${connectionName}.` : 'MongoDB connection successful.');
```